packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -137,8 +127,6 @@ contract PseZkVerifier is EssentialContract, IVerifier {
                 evidence.signalRoot,
                 evidence.graffiti,
                 prover,
-                blobHash,
```

Comment:
Dani, I think we still need `blobHash` and `txListHash` to be hashed into instnce.

Reply:
I think not necessary, because `metaHash` is there (and metaHash contains them (in blobHash variable hashed into it)).

Reply:
You still need to hash-in txListHash besides metahash.

Reply:
LiProposing.sol line 84: 
`bytes32 blobHash; //or txListHash (if Blob not yet supported)`
So `txListHash` is actually hashed into the metaHash - under the name `blobHash`.

Reply:
@Brechtpd am i wrong maybe here ? 🤔

Reply:
`pf.txListHash` still needs to be put into the instance because it's used to generate the random point, and so in the circuit we have to verify if it is indeed the correct value by hashing the data in the circuit.